docs/DeveloperGuide.md

```diff
@@ -449,37 +468,20 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     
     Use case ends.
 
-#### Use case: UC02 - Find customers
-
-**MSS:**
-
-1.  User requests to find customers by using keywords.
-2.  TrackBeau shows a list of customers with matching keywords.
-
-    Use case ends.
-
-**Extensions:**
-
-* 2a. The list is empty.
-
-  Use case ends.
-
-* *a. At any time, User requests to [view help UC01](#use-case-uc01---view-help)
-
-#### Use case: UC03 - List customers
+#### Use case: UC02 - List customers
```

Comment:
Can update the link to this use case? Previously was [list customers UC03](#use-case-uc03---list-customers) now becomes [list customers UC02](#use-case-uc02---list-customers)